component/main.jsonnet

```
@@ -162,8 +162,24 @@ local configmap = kube.ConfigMap(params.configMapName) {
   },
 };
 
+local secret = kube.Secret(params.secretName) {
+  metadata+: {
+    labels+: {
+      'app.kubernetes.io/name': params.secretName,
+      'app.kubernetes.io/instance': instanceName,
+      'app.kubernetes.io/component': 'fluent-bit',
+      'app.kubernetes.io/managed-by': 'commodore',
+    },
+  },
+  stringData: {
+    [s]: params.secret[s]
+    for s in std.objectFields(params.secret)
+  },
```

Hide `data` so Kubernetes is happy:

```suggestion
  },
  data:: {}
```